inference-engine/src/inference_engine/src/ie_core.cpp

```diff
@@ -116,18 +118,18 @@ template <typename F>
 void allowNotImplemented(F&& f) {
     try {
         f();
-    } catch (const InferenceEngine::NotImplemented&) {
+    } catch (const ie::NotImplemented&) {
     }
 }
 
-class CoreImpl : public InferenceEngine::ICore, public std::enable_shared_from_this<InferenceEngine::ICore> {
-    mutable std::map<std::string, InferenceEngine::InferencePlugin> plugins;
```

Comment:
can we remove `InferenceEngine::InferencePlugin`?

Reply:
We have to solve exceptions from DLL safety manually in each place where the `IInferencePlugin` API is used. It is very error-prone. With `InferencePlugin` we just keep all these wappers in a single place.

Reply:
We have `ov::InferencePlugin` and `Ie::InferencePlugin`. can we remove the last one?